content/docs/hello-world.md

```diff
@@ -17,7 +17,7 @@ ReactDOM.render(
 
 It displays a heading saying "Hello, world!" on the page.
 
-[](codepen://hello-world)
+**[Try it on CodePen](https://codepen.io/aniruddha-shriwant/pen/ExwqNQE?editors=0011)**
```

Comment:
Can we link to @gaearon's codepens? All codepens in this site link to his account.

Reply:
Ya sure, I will check his CodePen profile to see if there are any CodePens

Reply:
@harish-sethuraman is there any way to search from all these CodePen links? 
https://codepen.io/gaearon/pens/

Reply:
Or another solution would be, @gaearon himself updating these links and fixing this issue.

Reply:
> @harish-sethuraman is there any way to search from all these CodePen links? https://codepen.io/gaearon/pens/

I have got the link for one CodePen which was of "Hello World"
https://codepen.io/gaearon/pen/rrpgNB

Reply:
> is there any way to search from all these CodePen links?
I'm trying to figure out this as well :(

Reply:
> Can we link to @gaearon's codepens? All codepens in this site link to his account.

Also, I apologize for this, I wasn't knowing that all React Documentation uses his links :)

Reply:
Any updates? Need help?

Reply:
I apologize but can't do anything on this one as I wasn't able to find his CodePen links from his profile,
Will need @gaearon's help :)

Reply:
@harish-sethuraman do you have any other solution for this one?

Reply:
I'll try to find those codepen links. (I'll have to manually find them as well 😅 )

PS: It would be nice if you'd not mention name in every comment. mentioning sends a separate notification and replying in conv does the same

Reply:
> I'll try to find those codepen links. (I'll have to manually find them as well 😅 )

Sure :)

> PS: It would be nice if you'd not mention name in every comment. mentioning sends a separate notification and replying in conv does the same


Ohhh...So sorry for this, thanks for pointing this out, otherwise I would have done the same thing in all my future open source work that I'm planning to do xD

Reply:
Update: 
https://codepen.io/gaearon/pens/public
I had explored all the unnamed code pens from this link ( by unnamed I mean CodePen with titles like "gxNVDp" and so on) and wasn't able to find CodePens appropriate for these two files: 
1. content/docs/introducing-jsx.md
2. content/docs/rendering-elements.md

It looks like we will have to create new CodePen links for them

Reply:
Found em :) Once cross check if the codepens i've added is correct alone :)

Reply:
Ya wait, I'm checking them now